protocol/messenger_communities.go

```diff
@@ -1308,6 +1308,19 @@ func (m *Messenger) CancelRequestToJoinCommunity(request *requests.CancelRequest
 		return nil, err
 	}
 
+	if !community.AcceptRequestToJoinAutomatically() {
+		// send cancelation to community admins also
```

Comment:
We were not sending the cancellation to the admins, so they kept the request to join.